content/finalist/ttjc@finalist@digvijaysrathore/index.md

```diff
@@ -0,0 +1,123 @@
+
+![image](https://avatars3.githubusercontent.com/u/59700645?s=460&u=59aaf467f690a7216b807c927c9b4b399657d98a&v=4)
+
+## why are you taking this challenge? 🤷
+
+I need an internship/job to have some experience of writing code for a porduction level application that helps a large number of users, which will ultimately let me learn and explore more in my niche.
+
+## something interesting about you? 🤩
+
+As a 15 year old I always think my next project will be a billion dollar startup.
+
+### how to contact me? 📮
+
+- [Twitter](https://twitter.com/novadigvijay)
+- [LinkedIn](https://www.linkedin.com/in/digvijaysrathore)
+- [Email](dynamicdigvijay@gmail.com)
+
+### projects 🧑‍💻
+
+- ### 1. Slash Co    
+
+    A platform for devs to share knowledge, comment & upvote posts, maintain a user profile. Developers can use markdown to write customized articles, add a picture, enable comments on posts. It is similar to dev.to.
+
+    **_Stacks_** `React`, `Node.js`, `FireBase`, `Neomorphic-UI`
+
+    - [**_hosted link_**](https://slashco.netlify.app/)    
+    - [**_github link_**](https://github.com/digvijaysrathore/slash-frontend/) 
+
+- ### 2. MERN Blog    
+
+    This MERN blog has medium-like rich text editor and user roles. Only certain user IDs are allowed to visit the admin pages and functionalities. I used CK editor to build the text editor and react-html-parser to parse the data.
+
+    **_Stacks_** `React`, `Node.js`, `CK Editor`
+
+    - [**_hosted link_**](https://theindialab.netlify.app/)    
+    - [**_github link_**](https://github.com/digvijaysrathore/mern-blog/) 
+
+- ### 3. Groot   
+
+    Groot connects farmers/gardners to urban people who are willing to buy organic veges. It's a directory of gardens around you. Join, add your farm, explore other farms and exchange veges. It's a full stack MERN application.
+
+    **_Stacks_** `React`, `Node.js`, `FireBase | FB Auth`, `Ant Design`
+
+    - [**_hosted link_**](https://grootconnect.netlify.app/)    
+    - [**_github link_**](https://github.com/digvijaysrathore/groot-frontend/) 
+
+- ### 4. Code Cafe   
+
+    Code Cafe is a tinder for coders. Fill in the form, join the platform, have some coffee and collaborate with devs. I used Redux for state management. It was my first project, I didn't built a custom backend and firebase instead. 
+
+    **_Stacks_** `React`, `Redux`, `FireBase`
+
+    - [**_hosted link_**](https://novacafe.netlify.app/)    
+    - [**_github link_**](https://github.com/digvijaysrathore/codecafe/) 
+
+- ### 5. Power List  
+
+    I built this productivity app while learning React Native. I used expo and generated an APK as well. Users can add tasks for the day and delete them upon completion. Find the APK on it's github repository.
+
+    **_Stacks_** `React Native`
+
+    - [**_hosted link_**](https://github.com/digvijaysrathore/powerlist)   
```

Comment:
can you release the **apk** and share the link or share the expo link after publishing.